.env.test

```diff
@@ -1,2 +1 @@
-SCOPE_API_ENDPOINT=<YOUR_SCOPE_API_ENDPOINT>
-SCOPE_APIKEY=<YOUR_SCOPE_API_KEY>
+SCOPE_API_ENDPOINT=https://app.scope.dev
```

Comment:
this might more more confusing than helping. Maybe better to just inline as we do in cypress